packages/fxa-settings/src/components/App/RememberPassword/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useCallback } from 'react';
```

Comment:
Nit: Missing license here.

Also for the `const linkTarget` and `const queryParam`, same comment around not needing to explicitly set types on those. 🙂